.gitmodules

```diff
@@ -1,30 +1,30 @@
 [submodule "aws-common-runtime/aws-c-common"]
```

Comment:
Didn't change this for readability?

Reply:
Didn't change it because it's just a name, and it would break the history.